hardware/README.rst

```diff
@@ -15,6 +15,109 @@ To find out how to join the Hardware SIG `get in touch. <https://www.oneapi.io/c
 Meeting Notes
 =============
 
+2023-06-22
+=========
+
+Agenda
+------
+
+* Consistent timestamp reporting in Level Zero, Matias Cabral
+* Level Zero port for TornadoVM + the SPIR-V code gen, Juan Fumero
+* Tiles-as-devices model in Level Zero, Jaime Arteaga
+* Level Zero and Unified Runtime general updates, Jaime Arteaga Molina, Alastair Murray
+
+Attendees
+---------
+
+.. list-table::
+
+
+   * - Alison L Richards, Intel
+     - Jaime Arteaga Molina, Intel
+     - Michal Mrozek, Intel
+   * - Alastair Murray, Codeplay
+     - Brice Videau, Intel
+     - Matias Cabral, Intel
+   * - Juan Fumero, The University of Manchester
+     - Kenneth Benzie, Codeplay
+     - Ben Ashbaugh, Intel
+   * - Brice Goglin
+     - John Daniel Holmes, Intel
+     - Ruyman Reyes, Codeplay
+   * - Kevin Harms, ANL
+     - Xinmin Tian, Intel
+     - Rod Burns, Codeplay
+   * - Brandon Yates, Intel
+     - DorotheeX Marie Clotilde Balas, Intel
+     -
+
+Consistent timestamp reporting in Level Zero
+--------------------------------------------
+
+* Inconsistency of metrics units for timestamps in
+  Level Zero specification.
+* Proposal: Explicit definition of timestamps on ns
+  and resolutions in frequency.
+
+Level Zero port for TornadoVM + the SPIR-V code gen
+---------------------------------------------------
+
+* Work on extending TornadoVM to run on oneAPI stack,
+  focusing on Level Zero.
+* Current acces to heterogeneous systems is fragmented
+  with different stack for CPU and GPU.
+* What if access to these hardware could be done from
+  existing high-level programming languages.
+* TornadoVM translates for instance Java Bytecode
+  into CUDA, OpenCL, or SPIRV.
+* Translation is done by using ACC annotations.
+* TaskGraphs methods are used to identify functions
+  to accelerate.
+* Found that for some workloads, suggested group size
+  returned by L0 driver is not as optimal as the one
+  found heuristically by application.
+* Garbage Collector needs to be controlled to avoid
+  failures when dealing with memory moved to an accelerator.
+* Some suggestions for L0:
+  * Migration counters.
+  * Async exception support.
+  * Device aggregation.
+  * Improvements to suggested group size returned by L0.
```

Comment:
```suggestion
  * Improvements to suggested group size returned by L0.
* Question: Unified runtime would be more useful to this project as a standard?  Answer: To convince Java community to use this software stack a standard is more appealing.  Easier to justify a standard because not controlled by one party.
```

Reply:
thanks @alycm . Added.